security/firely-auth/firely-auth-deploy.rst

```diff
@@ -137,17 +137,23 @@ See the instructions on :ref:`running Firely Server in Docker <use_docker>` to l
 Firely Auth is configured in the same way.
 
 
-
-
 .. _firely_auth_deploy_inmemory:
 
 InMemory user store
 -------------------
 
-The InMemory user store is only meant for testing your setup or evaluating Firely Auth.
-For production use configure the SQL Server user store.
+The InMemory user store is not supported since version 4.0. You will have to set up a Sqlite or a SqlServer database and add your users in the UI or through the :ref:`firely_auth_mgmt`.
+
+.. _firely_auth_deploy_sqlite:
+
+Sqlite user store
+-----------------
+
+Sqlite is setup by default and will create a database in the ./data/ folder. If you want to change this, you can alter the settings as described in :ref:`firely_auth_settings_userstore`
+
+To add users to the store, you can use the UI  or through the :ref:`firely_auth_mgmt` once the application has started.
 
-The users for the InMemory user store can be configured in :ref:`firely_auth_settings_userstore`
+When you want to use Sqlite as user store in docker, you will have to create a database file, mount it to your docker container, and adjust the Sqlite connectionstring.
```

Comment:
Could you add a link to the database settings here?

Reply:
done